src/rustdoc/resources.md

```diff
@@ -0,0 +1,14 @@
+# Resources
+There are various resources which are useful to contributors and team members.
+
+- [rustdoc internals](https://rustc-dev-guide.rust-lang.org/rustdoc-internals.html)
+- [rustc Developer Guide][dev_guide]
+  - Documentation on compiler internals and setting up a developer environment.
+- [rustc Generated Documentation][rustc]
```

Comment:
Should this say / also mention "rustdoc" and (also) directly link to `librustdoc`'s API docs?